packages/ai-constructs/src/conversation/conversation_handler_construct.ts

```diff
@@ -105,5 +125,23 @@ export class ConversationHandlerFunction
         ) as CfnFunction,
       },
     };
+
+    this.storeOutput(this.props.outputStorageStrategy);
   }
+
+  /**
+   * Append conversation handler to defined functions.
+   */
+  private storeOutput = (
+    outputStorageStrategy:
+      | BackendOutputStorageStrategy<FunctionOutput>
+      | undefined
+  ): void => {
+    outputStorageStrategy?.appendToBackendOutputList(functionOutputKey, {
+      version: '1',
+      payload: {
+        definedFunctions: this.resources.lambda.functionName,
```

Comment:
But this is not a `definedFunction`. We should store this under it's own AIConstructOutputKey.